infrastructure/terraform/modules/cognito-triggers/module_pre_token_generation_lambda.tf

```diff
@@ -0,0 +1,66 @@
+module "pre_token_generation_lambda" {
+  source = "../../../../../nhs-notify-shared-modules/infrastructure/modules/lambda"
```

Comment:
Need to update this once https://github.com/NHSDigital/nhs-notify-shared-modules/pull/86/files is merged and published